python/tests/unit/prompt_template/test_prompt_template_e2e.py

```diff
@@ -143,11 +154,15 @@ async def test_it_handle_edge_cases(self, kernel: Kernel, template: str, expecte
         if expected_result.startswith("ERROR"):
             with raises(TemplateSyntaxError):
                 await KernelPromptTemplate(
-                    prompt_template_config=PromptTemplateConfig(name="test", description="test", template=template)
+                    prompt_template_config=PromptTemplateConfig(
+                        name="test", description="test", template=template, allow_unsafe_content=True
+                    )
                 ).render(kernel, KernelArguments())
         else:
             result = await KernelPromptTemplate(
-                prompt_template_config=PromptTemplateConfig(name="test", description="test", template=template)
+                prompt_template_config=PromptTemplateConfig(
+                    name="test", description="test", template=template, allow_unsafe_content=True
+                )
             ).render(kernel, KernelArguments())
```

Comment:
Also need to test scenarios where unsafe content is disallowed: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel.UnitTests/PromptTemplate/KernelPromptTemplateTests.cs#L522-L951